client/ayon_core/tools/workfiles/widgets/save_as_dialog.py

```diff
@@ -172,6 +178,8 @@ def __init__(self, controller, parent):
         inputs_layout.addWidget(extension_combobox, 2, 1)
         inputs_layout.addWidget(preview_label, 3, 0)
         inputs_layout.addWidget(preview_widget, 3, 1)
+        inputs_layout.addWidget(artist_note_label, 4, 0)
+        inputs_layout.addWidget(artist_note_input, 4, 1)
```

Comment:
I have small UI comment that can be ignored: I would keep label and input on separate rows
```suggestion
        inputs_layout.addWidget(artist_note_label, 4, 0, 1, 2)
        inputs_layout.addWidget(artist_note_input, 5, 0, 1, 2)
```

Reply:
That makes it:

![image](https://github.com/user-attachments/assets/06b52cc1-ff71-4c3f-a1e4-bfd1189939ed)

Fine with me. @LiborBatek thoughts?